packages/components/src/toolbar/stories/index.tsx

```diff
@@ -57,13 +65,13 @@ function InlineImageIcon() {
 	);
 }
 
-const Template = ( props ) => (
+const Template: ComponentStory< typeof Toolbar > = ( props ) => (
 	<div style={ { height: 280 } }>
 		<Toolbar { ...props } />
 	</div>
 );
 
-export const Default = Template.bind( {} );
+export const Default: ComponentStory< typeof Toolbar > = Template.bind( {} );
```

Comment:
```suggestion
export const Default = Template.bind( {} );
```

Just a tip: These stories don't need type annotations as long as `Template` is typed 😎

Reply:
Great catch! I just missed that one even though the typing for `Template` was right there.